Examples/Algorithms/Geant4/CMakeLists.txt

```diff
@@ -9,9 +7,13 @@ add_library(
   src/MMSteppingAction.cpp)
 target_include_directories(
   ActsExamplesGeant4
-  PUBLIC
-    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
-    ${Geant4_INCLUDE_DIRS})
+  PUBLIC ${Geant4_DEFINITIONS})
```

Comment:
Shouldn't that be a `target_compile_definitions`?

(Just looking at changes to other CMake files, don't mind me if I'm just showing my CMake ignorance.)

Reply:
Yes, it should. Thanks for catching that. I has been fixed now.